docs/ops.html

```diff
@@ -1662,157 +1661,189 @@ <h5><a id="kafka_streams_thread_monitoring" href="#kafka_streams_thread_monitori
+        <td>The total number of punctuate calls.</td>
+        <td>kafka.streams:type=stream-thread-metrics,thread-id=([-.\w]+)</td>
       </tr>
       <tr>
         <td>task-created-rate</td>
-        <td>The average number of newly created tasks per second.</td>
-        <td>kafka.streams:type=stream-metrics,client-id=([-.\w]+)</td>
+        <td>The average number of tasks created per second.</td>
+        <td>kafka.streams:type=stream-thread-metrics,thread-id=([-.\w]+)</td>
       </tr>
       <tr>
         <td>task-created-total</td>
         <td>The total number of tasks created.</td>
-        <td>kafka.streams:type=stream-metrics,client-id=([-.\w]+)</td>
+        <td>kafka.streams:type=stream-thread-metrics,thread-id=([-.\w]+)</td>
       </tr>
       <tr>
         <td>task-closed-rate</td>
         <td>The average number of tasks closed per second.</td>
-        <td>kafka.streams:type=stream-metrics,client-id=([-.\w]+)</td>
+        <td>kafka.streams:type=stream-thread-metrics,thread-id=([-.\w]+)</td>
       </tr>
       <tr>
         <td>task-closed-total</td>
         <td>The total number of tasks closed.</td>
-        <td>kafka.streams:type=stream-metrics,client-id=([-.\w]+)</td>
-      </tr>
-      <tr>
-        <td>skipped-records-rate</td>
-        <td>The average number of skipped records per second.</td>
-        <td>kafka.streams:type=stream-metrics,client-id=([-.\w]+)</td>
-      </tr>
-      <tr>
-        <td>skipped-records-total</td>
-        <td>The total number of skipped records.</td>
-        <td>kafka.streams:type=stream-metrics,client-id=([-.\w]+)</td>
+        <td>kafka.streams:type=stream-thread-metrics,thread-id=([-.\w]+)</td>
       </tr>
  </tbody>
 </table>
 
 <h5><a id="kafka_streams_task_monitoring" href="#kafka_streams_task_monitoring">Task Metrics</a></h5>
-All the following metrics have a recording level of <code>debug</code>:
+All the following metrics have a recording level of <code>debug</code>,
+except for metrics dropped-records-(rate | total) which have a recording level
+of <code>info</code>:
  <table class="data-table">
       <tbody>
       <tr>
         <th>Metric/Attribute name</th>
         <th>Description</th>
         <th>Mbean name</th>
       </tr>
+      <tr>
+        <td>process-latency-avg</td>
+        <td>The average execution time in ns, for processing.</td>
+        <td>kafka.streams:type=stream-task-metrics,thread-id=([-.\w]+),task-id=([-.\w]+)</td>
+      </tr>
+      <tr>
+        <td>process-latency-max</td>
+        <td>The maximum execution time in ns, for processing.</td>
+        <td>kafka.streams:type=stream-task-metrics,thread-id=([-.\w]+),task-id=([-.\w]+)</td>
+      </tr>
+      <tr>
+        <td>process-rate</td>
+        <td>The average number of processed records per second across all source processor nodes of this task.</td>
+        <td>kafka.streams:type=stream-thread-metrics,thread-id=([-.\w]+),task-id=([-.\w]+),processor-node-id=all</td>
```

Comment:
This should be `stream-task-metrics` right?

Also do you remember what's the rationale of adding the `processor-node-id=all` tag, could we just drop this from this task sensor -- I know it maybe too late to drop, but wondering why we needed this tag in the map.

Reply:
Good catch!

I guess we have that tag because the metric is a roll-up metric which measures across all processor nodes. I took over the tag from the previous built-in metrics version. 

I removed the tag from the docs.

Reply:
Actually in the source code we do add the `processor-node-id=all` so we cannot just remove it from the docs -- what I was asking is, if we can remove this tag in the code, so that we can remove it in the docs -- anyways we need to make the doc consistent with code, which currently add this tag to the map, and I'm wondering if it is necessary or can be removed (in both code and then docs).

Reply:
Currently, I do not see a reason we would need this tag. However, I have to try to remove it to be completely sure.

Reply:
Cool, could you try removing that in the source code and then see if that it still okay from unit tests?